{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -1,19 +1,38 @@
-FROM python:3.6
+{% if cookiecutter.use_alpine_linux == "y" -%}
+ARG BASE_IMAGE=python:3.6-alpine
+{% else -%}
+ARG BASE_IMAGE=python:3.6
+{% endif -%}
 
-COPY ./bin/s6-overlay-amd64.tar.gz /tmp/s6-overlay-amd64.tar.gz
+FROM $BASE_IMAGE AS base-image
 
-RUN tar xvfz /tmp/s6-overlay-amd64.tar.gz -C / \
-    && rm -f /tmp/s6-overlay-amd64.tar.gz {% if cookiecutter.use_celery == "y" %}\
-    && adduser --disabled-password --gecos '' celery{% endif %}
+{% if cookiecutter.use_alpine_linux == "y" -%}
+RUN apk add postgresql-dev gcc python3-dev musl-dev
+{% endif -%}
 
-COPY ./src/ /srv/app/
+ENV PYTHONUNBUFFERED=1
+ENV PATH=/root/.local/bin:$PATH
```

Comment:
most Dockerfile instructions, including ENV vars, are scoped per build-stage, and will not be preserved in the next stage

Reply:
right